grails-app/i18n/messages.properties

```diff
@@ -2737,5 +2738,64 @@ react.stockListManagement.emailSubject.label=STOCK LIST UPDATE
 react.stockListManagement.alert.emailSend.label=Email sent successfully
 react.stockListManagement.alert.noManagerAssociated.label=There is no manager associated with this stock list. Please add a manager and try again.
 react.stockListManagement.alert.noManagerSelected.label=Please add a manager as a recipient and resend.
+
+# Dashboard titles
+react.dashboard.numberData.inventoryByLotAndBin=Inventory Details by Lot and Bin
```

Comment:
Please add the `.label` at the end of each property key to match the pattern.

Reply:
ok !